src/main/java/de/tum/in/www1/artemis/service/dto/AbstractBuildResultNotificationDTO.java

```diff
@@ -0,0 +1,24 @@
+package de.tum.in.www1.artemis.service.dto;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+public abstract class AbstractBuildResultNotificationDTO {
+
+    public abstract ZonedDateTime getBuildRunDate();
+
+    public abstract Optional<String> getCommitHashFromAssignmentRepo();
+
+    public abstract Optional<String> getCommitHashFromTestsRepo();
+
+    public abstract boolean isBuildSuccessful();
+
+    public abstract Long getBuildScore();
```

Comment:
Do we actually need this method? The JenkinsDTO and BambooDTO behave differently at the moment but the actual score calculation is done somewhere else. We should just set a placeholder 0L and delete this method.

Reply:
The reason I added that is because I wasn't sure: in Bamboo the score was set to `0L`. In Jenkins, the score is calculated. Logically, it doesn't make sense to calculate scores in the services themselves, but I'm not sure if it's safe to remove.